docs/releases/release-1.4/release-team.md

```diff
@@ -0,0 +1,14 @@
+# Kubeflow 1.4 Release Team
+
+| **Role** | **Name** (**GitHub / Slack ID**) |
+|----------|----------------------------------|
+| Release Manager | Kimonas Sotirchos (GitHub: [@kimwnasptd](https://github.com/kimwnasptd) / Slack: `@kimwnasptd`) |
+| Release Team Member(s) | David van der Spek (GitHub: [@DavidSpek](https://github.com/DavidSpek) / Slack: `@David van der Spek`), Malini Bhandaru (GitHub: [@mkbhanda](https://github.com/mkbhanda) / Slack: `@Malini Bhandaru`), and Rui Vasconcelos (GitHub: [@RFMVasconcelos](https://github.com/RFMVasconcelos) / Slack: `@Rui Vasconcelos`) |
+| Product Manager |  |
+| Docs Lead | Rui Vasconcelos (GitHub: [@RFMVasconcelos](https://github.com/RFMVasconcelos) / Slack: `@Rui Vasconcelos`) |
+| Working Group Liaison(s) |  |
+| Distribution Representative |  |
+| Shadow(s) | Anna Jung (GitHub: [@annajung](https://github.com/annajung) / Slack: `@annajung`) |
```

Comment:
I believe there was one more person interested in being a shadow. @RFMVasconcelos could you help gather the info needed to be added here?